### ISSUE TITLE
Flair58 connector

### ISSUE THREAD

**Issue by frontmesh** [sybond/ble-pressure-transducer#1]
Hi @sybond , 

I've seen your post in some of the groups and was wondering to try this out with my Flair 58. 
Transducer which you purchased (as stated in README - https://www.aliexpress.com/item/1005004559608411.html) doesn't have proper flair 58 pressure thread connection. Did you have to buy some sort of adapter so that you were able to use it with your Flair 58? 
Another solution to prevent leaks from the thread would be to use sealant tape which should be put on to the thread every time you change the pressure device which I am trying to avoid. 

Thanks,
Vlad

**Comment by sybond** [comment]
> Did you have to buy some sort of adapter so that you were able to use it with your Flair 58?

No. Make sure you ordering G1/8. They mention _"Pressure connection: 1/8”NPT, Other pressure connection is on request"_.

> Another solution to prevent leaks from the thread would be to use sealant tape which should be put on to the thread every time you change the pressure device which I am trying to avoid.

Even though I have ordered the correct G1/8, I still use seal tape to prevent leaks. In my case, I'll stick to my pressure sensor, even if I don't record the data every time.

**Comment by frontmesh** [comment]
Thanks for the clarification. I have indeed received transducer with G1/8 connection. 
What I did not release is that there are two types (short and long) of G1/8 connections. 
Flair uses longer connection with a rubber ring at the top which prevents leaks
https://pressensor.com/products/fitting-for-compatibility-with-g1-8-connections
and a short one https://pressensor.com/products/fitting-for-compatibility-with-g1-8-short-connections
I have another digital Elitech PGW-500 transducer which also has short G1/8 connection.
